.github/workflows/old-compilers.yml

```
@@ -100,6 +101,7 @@ jobs:
             COMPILER: clang
             VERSION: 14
             STATS: OFF
+            BOOST: OFF
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)